chameleon/settings.py

```diff
@@ -319,6 +319,15 @@
 # Logger config
 #
 #####
+
+# DEBUG = True if DJANGO_ENV_DEBUG==DEBUG
```

Comment:
Did you mean to comment this out still?

Reply:
That was informational, not code. Replaced with links to the django debug setting docs.